rest/utilities_testing.go

```diff
@@ -220,17 +220,8 @@ func (rt *RestTester) Bucket() base.Bucket {
 	}
 	rt.TestBucket = testBucket
 
-	if rt.InitSyncSeq > 0 {
-		if base.TestsUseNamedCollections() {
-			rt.TB().Fatalf("RestTester InitSyncSeq doesn't support non-default collections")
-		}
-
-		log.Printf("Initializing %s to %d", base.DefaultMetadataKeys.SyncSeqKey(), rt.InitSyncSeq)
-		tbDatastore := testBucket.GetMetadataStore()
-		_, incrErr := tbDatastore.Incr(base.DefaultMetadataKeys.SyncSeqKey(), rt.InitSyncSeq, rt.InitSyncSeq, 0)
-		if incrErr != nil {
-			rt.TB().Fatalf("Error initializing %s in test bucket: %v", base.DefaultMetadataKeys.SyncSeqKey(), incrErr)
-		}
+	if rt.PersistentConfig != false {
+		require.Zero(rt.TB(), rt.InitSyncSeq, "RestTesterConfig.InitSyncSeq is not supported on rosmar")
```

Comment:
I still don't think we should or need to do this.

`InitSyncSeq` isn't related to persistent config and initializing a non-zero seq _should_ work just fine on Rosmar? I do take your point that the above skipped test is the only one using `InitSyncSeq` but I don't think that should be reason to add these restrictions.

Reply:
This is actually just a typo in the description, I just wanted to protect the caller from setting `InitSyncSeq` with `PersistentConfig: true` where this would be a no-op. Fixed the text here.